<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLSLiveInfoAfreecaTV</class>
 <widget class="QWidget" name="PLSLiveInfoAfreecaTV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>19</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="PLSLiveInfoDualWidget" name="dualWidget" native="true"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="midwidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="scheduleLabel">
        <property name="text">
         <string>LiveInfo.base.Title</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="PLSLineEdit" name="lineEditTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="liveTitleLabel">
        <property name="text">
         <string>LiveInfo.Base.Category</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PLSLineEdit" name="lineEditCategory">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="tipLabel">
        <property name="text">
         <string>LiveInfo.live.afreecatv.category.detail</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QWidget" name="lineView" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="bottomButtonWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>493</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="useFor" stdset="0">
            <string notr="true">Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
           <property name="useFor" stdset="0">
            <string notr="true">Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PLSLineEdit</class>
   <extends>QLineEdit</extends>
   <header>PLSEdit.h</header>
  </customwidget>
  <customwidget>
   <class>PLSLiveInfoDualWidget</class>
   <extends>QWidget</extends>
   <header>PLSLiveInfoBase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
